pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[build-system]
+requires = [
+  "hatch-vcs>=0.3",
+  "hatchling>=1.12.2",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "eventlet"
+authors = [
+    {name = "Liden Lab", email = "eventletdev@lists.secondlife.com"},
+]
+description = "Highly concurrent networking library"
+readme = "README.rst"
+requires-python = ">=3.8"
+license = {text = "MIT"}
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python",
+    "Topic :: Internet",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+dynamic = ["version"]
+dependencies = []
+
+[project.urls]
+Homepage = "https://github.com/eventlet/eventlet"
+History = "https://github.com/eventlet/aiohub/blob/main/NEWS"
+Tracker = "https://github.com/eventlet/eventlet/issues"
+Source = "https://github.com/eventlet/aiohub"
+
+[project.optional-dependencies]
+dev = ["black", "isort", "pip-tools", "build", "twine", "pre-commit", "commitizen"]
```

Comment:
This is optional development dependencies.

If you are ok we that, once major issues will be solved, we could start using `pre-commit` and migrate to `black` formatting. Using `black` would avoid to loose time with formatting issues and pep8...